BACKLOG.md

```diff
@@ -24,7 +19,7 @@ the request and want to put work into it.
 - Status: Further investigation required. We already have
   some analysis in the form of custom Roslyn analyzers.
   
-## Storage client-side encryption
+## Storage: Client-side encryption
```

Comment:
Perhaps we should remove this one as well, given internal work?

Reply:
Ah yes, makes sense.